gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/v4/endpoint/EndpointResource.java

```diff
@@ -122,4 +122,27 @@ public PluginMoreInformation getMoreInformation(@PathParam("endpoint") String en
 
         return endpointService.getMoreInformation(endpointId);
     }
+
+    @GET
+    @Path("sharedConfigurationSchema")
```

Comment:
In our guidelines for the v4 rest API, we choose to use kebab case for the path: https://gravitee.slab.com/posts/gravitee-rest-api-guidelines-8ta6qkxl#h0i2u-case

```suggestions
    @Path("shared-configuration-schema")
```

Reply:
good catch